packages/contracts/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Created the `IMultisig` interface.
+- Adds check to `Multisig.initialize()` that members length isn't bigger than max uint16.
```

Comment:
It seems like the changelog hasn't been maintained in the past. The upcoming version doesn't contain that many changes.

Reply:
you have the fix for it here #333